test/unit/calculators/benefit_cap_calculator_configuration_test.rb

```diff
@@ -112,6 +120,15 @@ class BenefitCapCalculatorConfigurationTest < ActiveSupport::TestCase
             assert_equal "other_first_question", @config.benefits(:other).fetch("first_benefit")["question"]
           end
         end
+        context "merge questions" do
+          should "get all benefit questions from multiple configuration sets" do
+            questions = @config.all_questions
+            assert_includes questions.values, "other_first_question"
+            assert_includes questions.values, "other_second_question"
+            assert_includes questions.values, "third_question"
+            assert_includes questions.values, "other_third_question"
```

Comment:
first_question and second_question are (and should be) overwritten by other_first_question and other_second_question as they are duplicates. The test is checking that we get all discreet questions from multiple configuration sets.

I have addressed this point by changing the test description in this commit: d374998165d392a7e1a6610b4d73e121453e34c8